packages/patternfly-3/patternfly-react/src/components/VerticalNav/VerticalNavItemHelper.js

```diff
@@ -82,18 +82,15 @@ class BaseVerticalNavItemHelper extends React.Component {
 
   onItemClick = event => {
     const { primary, secondary, tertiary } = this.getContextNavItems();
-    const { isMobile, updateNavOnItemClick, idPath } = this.props;
-    const { href, onClick } = this.navItem();
-    event.preventDefault();
+    const { isMobile, preventHref, updateNavOnItemClick, idPath } = this.props;
+    const { onClick } = this.navItem();
+    if (preventHref && !!onClick) event.preventDefault();
```

Comment:
Could you format this with the `then` in a new block?

Reply:
if asked to not prevent using the href, why prevent it when there is an onClick?

Reply:
If asked to not prevent (`preventHref: false`) then it won't be prevented.
Because `preventHref` is default as `true`, I want to make sure there is an onClick prop aswell so if the `href` is prevented there is a function to call